iceberg/iceberg-shading/pom.xml

```diff
@@ -121,7 +117,6 @@
                                             <include>org.apache.parquet:*</include>
                                             <include>com.google*:*</include>
                                             <include>com.fasterxml*:*</include>
-                                            <include>com.github.ben-manes*:*</include>
```

Comment:
Are we sure we want to remove it from the fat jar? will the engines be able to get the classes from somewhere else?

Reply:
Fixed. Good catch!